src/executor/stack/executor.rs

```diff
@@ -574,24 +568,22 @@ impl<'config, 'precompiles, S: StackState<'config>, P: PrecompileSet>
 		value: U256,
 		init_code: Vec<u8>,
 		gas_limit: u64,
-		access_list: Vec<(H160, Vec<H256>)>, // See EIP-2930
+		access_list: &[(H160, Vec<H256>)], // See EIP-2930
 	) -> (ExitReason, Vec<u8>) {
-		let address = self.create_address(CreateScheme::Fixed(address));
-
 		event!(TransactCreate {
 			caller,
 			value,
 			init_code: &init_code,
 			gas_limit,
-			address
+			address: self.create_address(CreateScheme::Fixed(address)),
 		});
 
 		if let Err(e) = self.record_create_transaction_cost(&init_code, &access_list) {
 			return emit_exit!(e.into(), Vec::new());
 		}
 
 		// Initialize initial addresses for EIP-2929
-		self.initialize_addresses(caller, address, access_list);
+		self.initialize_addresses(&[caller], access_list);
```

Comment:
If you try to compile the code with the `create-fixed` features, you will get a compile error here because the `initialize_addresses` accepts `Vec<(H160, Vec<H256>)` rather than `&[(H160, Vec<H256>)]`. Overall, it looks like it isn't worth changing the `Vec` to slice here.

Reply:
Added to CI.